### PR TITLE
[#9536] Feedback MCQ/MSQ Question E2E Test

### PR DESCRIPTION
**Pull request by jtankw3**
Part of #9536

**Outline of Solution**

- Add backdoor method to get FeedbackResponse
- Edit page functions:
    - verify loaded question
    - add new question
    - copy question
    - edit question
- Submit page functions:
    - verify loaded question
    - submit response
    - check previous response
    - edit response

### CODE REVIEW

File: src/e2e/java/teammates/e2e/pageobjects/InstructorFeedbackEditPage.java [hunk 1]
```diff
@@ -862,4 +921,251 @@ private void selectVisibilityDropdownOption(int questionNum, String text) {
...
+
+        inputOptions(questionNum, questionDetails.getMsqChoices());
+        inputOptionWeights(questionNum, questionDetails.hasAssignedWeights(), questionDetails.getMsqWeights());
+        inputOtherChoice(questionNum, questionDetails.isOtherEnabled(), questionDetails.getMsqOtherWeight());
+        inputMaxOptions(questionNum, questionDetails.getMaxSelectableChoices());
+        inputMinOptions(questionNum, questionDetails.getMinSelectableChoices());
+    }
+
+    private void inputMaxOptions(int questionNum, int maxOptions) {
+        if (maxOptions == Integer.MIN_VALUE) {
```
Comment (rrtheonlyone) [inline src/e2e/java/teammates/e2e/pageobjects/InstructorFeedbackEditPage.java:1155]: I notice this a few times here. This looks pretty hacky - isnt there a null state for the max/min objects within the object itself?

It looks we set it as `Integer.MIN_VALUE` in our E2E test to leave it unchecked.

**Review by rrtheonlyone — CHANGES_REQUESTED** [review]
Nice! The tests look good 💯 

For MCQ:
- testing of adding/removing options?
- shall we add in more students to course? I feel like the generated options feature is not really being tested with only student

For MSQ:
- testing the maximum/minimum number of options that can be selected? we leave it blank at the moment
- testing of leaving other option box blank with 'None of the above'

File: src/e2e/java/teammates/e2e/pageobjects/InstructorFeedbackEditPage.java (hunk 1 above)
Comment (jtankw3) [reply src/e2e/java/teammates/e2e/pageobjects/InstructorFeedbackEditPage.java:1155]: `Integer.MIN_VALUE` is the actual value being used to represent max and min option are not selected in `FeedbackMsqQuestionDetails` class

**Comment by jtankw3** [comment]
> Nice! The tests look good 💯
> 
> For MCQ:
> 
> * testing of adding/removing options?
> * shall we add in more students to course? I feel like the generated options feature is not really being tested with only student
> 
> For MSQ:
> 
> * testing the maximum/minimum number of options that can be selected? we leave it blank at the moment
> * testing of leaving other option box blank with 'None of the above'

Hi, once again thank you for your review.

- The adding/removing of options is tested under "edit question" test case where different choices are passed to `editMcqQuestion` and so new choices are added and old choices are removed.
- There are actually 5 students in the course. `student` in the test case just represents the student doing the submission.
- For the `loadedQuestion`, the `maxSelectableChoice` is actually 2. I did not test selecting more than 2 as I believe negative testing should not be done in E2E.
- When you click none of the above, all other options cannot be selected. However, you are right I did not test selecting none of the above. I will edit the "edit response" test case to choose that instead.

**Review by rrtheonlyone — APPROVED** [review]
LGTM 👍

File: src/e2e/java/teammates/e2e/pageobjects/InstructorFeedbackEditPage.java (hunk 1 above)
Comment (wkurniawan07) [reply src/e2e/java/teammates/e2e/pageobjects/InstructorFeedbackEditPage.java:1180]: @rrtheonlyone actually made a good point here. If `Integer.MIN_VALUE` has a special meaning, then it should be in its own variable.

**Review by wkurniawan07 — APPROVED** [review]
LGTM

**Pull request merged**